src/tests/ftest/util/recovery_utils.py

```diff
@@ -0,0 +1,154 @@
+"""
+  (C) Copyright 2024 Intel Corporation.
+  (C) Copyright 2025 Hewlett Packard Enterprise Development LP
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+import os
+import time
+
+from exception_utils import CommandFailure
+from run_utils import run_remote
+
+
+def get_vos_file_path(log, server_manager, pool):
```

Comment:
Why not define this under DaosServerManager too? It already has `log` and `pool` is just passed to `server_manager.get_vos_path`. So it could be used as
```
self.server_managers[0].get_vos_file_path(self.pool)
```

Reply:
I like that idea.  Implemented.